internal/context/amf_ue.go

```diff
@@ -134,7 +134,8 @@ type AmfUe struct {
 	/* Pdu Sesseion context */
 	SmContextList sync.Map // map[int32]*SmContext, pdu session id as key
 	/* Related Context */
-	RanUe map[models.AccessType]*RanUe
+	RanUe        map[models.AccessType]*RanUe
+	HoldingRanUe map[models.AccessType]*RanUe
```

Comment:
1. I think HoldingRanUe should support multiple RanUes per access type since more than 1 InitialUeMessage referring to the same AmfUe is possible.
2. If a malicious UE sends InitialUeMessage without following actions, as my understanding to this PR, the original amfUe.ranUe will be replaced with this malicious UE and the genuine UE will be stored into HoldingRanUe. Will this cause any problem? (e.g. Can AmfUe still handle the request from the genuine UE?)